modules/ipi-install-node-requirements.adoc

```diff
@@ -8,9 +8,9 @@
 
 Installer-provisioned installation involves a number of hardware node requirements:
 
-* *CPU architecture:* All nodes must use `x86_64` 
+* *CPU architecture:* All nodes must use x86_64 
 ifndef::openshift-origin[]
-or `arm64` 
+or AArch64
```

Comment:
Does this need backticks?

Reply:
Ah, thanks for linking this! Cool :) and relevant to a PR I currently have open!